app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!, only: %i[fight add_experience]
+
+  def fight
+    @user = current_user
+    @bot = @user.dup
+    @bot.hp = rand(@user.hp * 0.8..@user.hp + @user.hp * 0.2)
+  end
+
+  def update
+    @user = User.update_attribute(:experience, 52)
+    if @user.save
```

Comment:
`update_attribute` and then `save`?? :) Does not `update_attribute` do save?

Reply:
and ` User.update_attribute(:experience, 52)` -- which user will have updated experrience?

Reply:
def update will be removed. unused